### ISSUE TITLE
Make the span active after extraction within the jersey filter

### ISSUE THREAD

**Issue by hloos** [jaegertracing/jaeger-client-java#204]
Hello,

Maybe I misunderstand the way the span needs to be used but, in jaeger-jaxrs2, when using the com.uber.jaeger.filters.jaxrs2.ServerFilter, the span created after extraction of the request headers is currently not made active within the tracer instance.

```
SpanContext spanContext = tracer.extract(Format.Builtin.HTTP_HEADERS, carrier);
if (spanContext != null) {
    builder = builder.asChildOf(spanContext);
}
Span serverSpan = builder.startManual();

// the following line is missing
tracer.makeActive(serverSpan);

traceContext.push(serverSpan);
```

Therefor, when building a new span within the application, a new span is created instead of a child of the one extracted from the request.

```
try(ActiveSpan activeSpan = tracer.buildSpan("jumpcargo-my-friend").startActive()) {
}
```

Version of opentracing: 0.30.0
Version of jaeger-client-java: 0.20.0

**Comment by pavolloffay** [comment]
Hi @hloos,

thanks for reporting it. There is a plan to deprecate instrumentations in this repository in favor of https://github.com/opentracing-contrib/. There is already instrumentation for JAX-RS https://github.com/opentracing-contrib/java-jaxrs which should solve this problem.

**Comment by hloos** [comment]
Ok, thanks for the answer, good to know :)

There is also [https://bm-artifacts.misys.global.ad/maven/index.html#nexus-search;quick~wrap-opentracing-web-servlet-filter](https://bm-artifacts.misys.global.ad/maven/index.html#nexus-search;quick~wrap-opentracing-web-servlet-filter) which better suits my needs.

**Comment by pavolloffay** [comment]
It seems that it has been fixed as part of #257
